controller/node_controller.go

```diff
@@ -46,6 +51,13 @@ const (
 	snapshotChangeEventQueueMax = 1048576
 )
 
+type kernelVersion struct {
```

Comment:
What's the patch version if version is `5.4.2-123-aws`?

Reply:
That would be `kernel=5, major=4, minor=2, patch=123`

Reply:
Removed the kernel check.